person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  # getters and setters
+  attr_accessor :name, :age
+  attr_reader :id
+
+  # constructor
+  def initialize(_id, age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
Great job @Rebelzob 🤝

- [ ] [Optional] Kindly maintain the equal sign used for consistency. Implement this across all constructors.
![sign](https://github.com/Rebelzob/oop-school-library/assets/63917725/a7df94d9-4e8d-4f76-a69f-9e990a39ee08)
 reasons.

Reply:
Hi @1GORDON, Rubocop suggested I use this notation instead of the equal sign 
![image](https://github.com/Rebelzob/oop-school-library/assets/52711805/5f897f3d-55c8-449e-aa44-bb0014b1b781)